### PR TITLE
Tentative Nix Fix

### PR DESCRIPTION
**Pull request by hivert**

### CODE REVIEW

File: .nix/config.nix [hunk 1]
```diff
@@ -7,7 +7,7 @@
   ## The attribute to build from the local sources,
   ## either using nixpkgs data or the overlays located in `.nix/coq-overlays`
   ## Will determine the default main-job of the bundles defined below
-  attribute = "coq-combi";
+  attribute = "Coq-Combi";
```
Comment (CohenCyril) [inline .nix/config.nix]: If you rename this, you need to rename the overlay
`.nix/coq-overlays/coq-combi/default.nix`
to
`.nix/coq-overlays/Coq-Combi/default.nix`
but then you are not respecting the current conventions of nixpkgs (cf https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/coq-packages.nix)

Comment (CohenCyril) [inline .nix/config.nix]:
```suggestion
  attribute = "coq-combi";
```

Comment (hivert) [reply .nix/config.nix]: @CohenCyril : Nix and CI seems to work now !!! Thanks a lot for debugging my mess !

Comment (hivert) [reply .nix/config.nix]: @CohenCyril I manage to get rid of the error, but no test is actually performed ! I purposedly inserted a stupid bla in https://github.com/math-comp/Coq-Combi/commit/389926250c65b0eb74257aaf0ba1a6bb5747e400 and the CI didn't complain... Sorry.

**Pull request merged**